src/sequence.rs

```diff
@@ -950,6 +998,38 @@ mod tests {
     assert_eq!(length_value(&b[..]), Err(Err::Incomplete(Needed::Size(5))));
   }
 
+  #[cfg_attr(rustfmt, rustfmt_skip)]
+  //#[test]
```

Comment:
Is this supposed to be commented out?

Reply:
Ah. I commented it out to get more compile errors while debugging. Didn't mean to commit it